src/fibonacci.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+export default function fibonacci (number) {
+var fibArray = [0,1]
```

Comment:
Try to use ES6 features - `let` instead of `var`, and `const` if you're not going to initialize the value of the variable after the first initialization.